examples/parallel_for/seismic/main.cpp

```diff
@@ -91,11 +99,17 @@ int main(int argc, char *argv[]) {
             std::cout << "Substituting 1000 for unlimited frames because not running interactively"
                       << "\n";
         }
+        if (options.numberofIterations == 0) {
+            options.numberofIterations = 10;
+            std::cout << "Setting the number of iterations = 10 default"
+                      << "\n";
+        }
```

Comment:
The utility::cli_argument_pack() doesn't have the ability to specify default. Added a TODO comment, since this change is not in the scope of this pull request.